### DISCUSSION TITLE
ERR_SSL_PROTOCOL_ERROR on "go" and "docs" subdomains

### DISCUSSION THREAD

**Discussion by SilentHaiku** [reqnroll/Reqnroll#217] [General]
20-July-2024 at 3:40 pm

I'm getting an SSL error when I go here:
     _[https://github.com/reqnroll](https://github.com/reqnroll)_
and click on either of these two links:
     _[QuickStart guide](https://go.reqnroll.net/quickstart) (https://go.reqnroll.net/quickstart)_
     _[Reqnroll Documentation](https://docs.reqnroll.net/) (https://docs.reqnroll.net/)_

I tried turning off my VPN. That didn't make any difference.
I tried using a different browser. That didn't make any difference.
**I suspect your SSL certs don't cover your "go" and "docs" subdomains.**

The full message is:
This site can’t provide a secure connection
docs.reqnroll.net sent an invalid response.
Try running Windows Network Diagnostics
ERR_SSL_PROTOCOL_ERROR

**Comment by ajeckmans** [comment]
They are working fine here. Do you have the latest lets encrypt  certificates? https://letsencrypt.org/2024/04/12/changes-to-issuance-chains

I've noticed that on some machines that are controlled by an organization the windows update that provided these did not correctly install (without any warning), which causes some issues with validation of the certificate.

**Reply by SilentHaiku** [reply]
I whitelisted the URLs in Malwarebytes. They're accessible now.
(Before that, I tried importing the R10 an R11 intermediate certs that were missing on my computer, but that didn't solve the problem.)

**Reply by gasparnagy** [reply]
Oh... than this is the good old issue https://github.com/reqnroll/Reqnroll/issues/79. It will never end. 

Thanks for submitting the whitelist request, this is the best what we can do...

**Comment by gasparnagy** [comment]
I close this topic, let's reopen if there is anything to add.
